### DISCUSSION TITLE
How to move favorites to another operating system?

### DISCUSSION THREAD

**Discussion by hardmagnett** [DISTRHO/Cardinal#481] [Q&A]
Hello.
I need to use another laptop for using Cardinal.

It is so long and borind task to check all these modules again.

How can I copy all my modules marked as favorites to anohter PC?

Maybe  to achieve it I need to copy some settings file or directory.

**Comment by falkTX** [comment]
favorites are per session. I didnt even want to have those enabled because it leads to situations like these...
anyhow, you can create a plugin preset via the host, and import that preset in another PC which will contain plugin internal data.

**Reply by hardmagnett** [reply]
Thanks for answer.
